src/main/resources/plugin.yml

```diff
@@ -0,0 +1,23 @@
+name: AdminTools
+version: @version@
```

Comment:
Where does this get filled in from?

Reply:
It pulls it from pom.xml when compiling.